sampleconfig/core.yaml

```diff
@@ -336,6 +336,27 @@ peer:
         #  - from:
         #    to:
 
+        # Config parameters for the Byzantine Fault Tolerant (BFT) client implementation
+        bft:
+            # Whether to use a BFT client implementation
+            enabled: false
```

Comment:
let's put this at enabled at default. Meaning - let's change this key to 'disabled' and make it 'false' by default.

Reply:
Discussed f2f, decided to discuss further and possibly change it later.